types.ts

```diff
@@ -168,6 +169,7 @@ export type SelfcareParty = {
   taxCode: string;
   zipCode: string;
   origin: string;
+  userRole: UserRole;
```

Comment:
Actually this information is not returned from the BE!
Be careful when add new property to model retrieved from BE, if the field doesn't exists, it will always be undefined:
https://github.com/pagopa/selfcare-onboarding-backend/blob/release-dev/app/src/main/resources/swagger/api-docs.json#L541

You have to ask to BE team to add this field before you can merge this